tpch/gen-backup-tag.bash.ticat

```diff
@@ -0,0 +1,6 @@
+help = generate backup tag name by data scale
```

Comment:
File name not right, should be gen-tag.bash.ticat
Help not right, this should only generate `bench.workload.tag`